drake/doc/ubuntu.rst

```diff
@@ -49,3 +54,9 @@ Update the symbolic link in MATLAB to point to the version that was installed ea
     sudo ln -s /usr/lib/gcc/x86_64-linux-gnu/4.9/libstdc++.so libstdc++.so.6
 
 When you are done with these platform-specific steps, return to :doc:`from_source` to complete and test your installation.
+
+Note that when you run drake commands from now on (including the
+ones in the linked instructions, such as `make` or `make test`),
+you must always precede them with
+`env CXX=g++-4.9 CC=gcc-4.9 PATH=/usr/lib/ccache:$PATH`,
```

Comment:
@amcastro-tri My experience is that users tend to shoot themselves in the foot by setting compiler flags via `.bashrc`.  I advise against it, and never do it myself.

I do think there's a separate PR that makes Drake easier to configure and use, but I'm not trying to do that here.  I am trying to document  the currently-required mechanism, so that the (lots of!) people currently trying improve Drake every day can actually invoke the build system in the first place, without having to ask for help.  I would like to get this PR merged as incremental progress, rather than a final answer.